### ISSUE TITLE
Interactive Remote Attestation Client:  Support Concurrent Invocations 

### ISSUE THREAD

**Issue by jul-sh** [project-oak/oak#2908]
At the present both the Rust and Java client can only send invocations in sequence. That is they have to wait for a response to the previous message before sending a new message to the server. 

This is fine for the remote attestation handshake, in which the new message depends on the response to the previous. However, after the handshake, a client may wish to send invocations in parallel. It would be nice to support that. 

Replay protection would not be impacted by this, as each message continues to be identified by its unique nonce.

**Comment by conradgrobler** [comment]
We are no longer using the interactive remote attestation, so this is no longer relevant.
